### DISCUSSION TITLE
Why not coredns

### DISCUSSION THREAD

**Discussion by tigerzhang** [IrineSistiana/mosdns#24] [General]
感谢作者开发这个有趣的项目，试用效果很好，实现方式也跟我个人期望的很接近。

有个疑问，感觉在 coredns 上做几个插件，也能达到类似的效果，想问一下作者，出于什么考虑没有直接在 coredns 上做二次开发。

**Comment by IrineSistiana** [comment]
coredns 的插件只能按配置好顺序依次执行,很难实现直观的 if do if goto.

**Reply by wen-long** [reply]
coredns 也不支持并发向后端转发吧
